wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/RobotContainer.java

```diff
@@ -57,10 +58,6 @@ public RobotContainer() {
    * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
-    // Also set default commands here
-    m_drivetrain.setDefaultCommand(new TeleopArcadeDrive(m_drivetrain,
-        () -> m_controller.getRawAxis(1),
-        () -> -m_controller.getRawAxis(2)));
```

Comment:
I think setting the default command to be the teleop driving is the behavior we want to demonstrate in the examples. By doing so, we make it so users can write semi-autonomous commands that can take control of the drivetrain and then hand it back when they are done.

Reply:
If I set this, autonomous mode will be in "teleop" mode once the command completes but the driver station will still say autonomous. I thought that would be confusing to the user. Let me know if you still want to change that.

Reply:
That's the behavior on a real robot too -- once the autonomous command has finished executing, the command scheduler will schedule the default command even if the field is still in "autonomous mode".

Reply:
Seems odd to me that we are providing examples which will allow teleop commands in autonomous mode, but given that this seems to be the standard, it makes sense to conform.

Reply:
For FRC, the driver station sets all joystick inputs to 0 when in autonomous (except for the 2019 driver station when autonomous was sandstorm).

Reply:
The field/DS switching to Autonomous mode is just a trigger to schedule the autonomous command. You could do the same thing by hooking up the auto command to a button on the gamepad for example

Reply:
Ok, I think I changed it to make everyone happy. I renamed the command to arcadeDrive, set it to default, commented on it, and removed setting the command during teleop. Let me know if there is anything else, otherwise I will resolve this. Thanks.